Sources/MediaNavigationBar.swift

```diff
@@ -0,0 +1,135 @@
+        return minButton
+    }()
+    
+    private var mediaTitleTextLabel :UILabel = {
+        var label = UILabel()
+        label.numberOfLines = 1
+        label.textColor = .white
+        label.font = UIFont(name: "SFProDisplay-Medium", size: 17)
+        label.text = NSLocalizedString("TITLE", comment: "Video Title")
+        label.isUserInteractionEnabled = false
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    lazy var chromeCastButton :UIButton = {
+        var chromeButton = UIButton(type: .system)
+        chromeButton.addTarget(self, action: #selector(toggleChromeCast), for: .touchUpInside)
+        chromeButton.setImage(UIImage(named: "renderer"), for: .normal)
+        chromeButton.tintColor = .white
+        chromeButton.translatesAutoresizingMaskIntoConstraints = false
+        return chromeButton
+    }()
+    
+    lazy var airPlayButton :UIButton = {
+        var airPlayButton = UIButton(type: .system)
+        airPlayButton.addTarget(self, action: #selector(toggleAirPlay), for: .touchUpInside)
+        airPlayButton.setImage(UIImage(named: "iconAirplay"), for: .normal)
+        airPlayButton.tintColor = .white
+        airPlayButton.translatesAutoresizingMaskIntoConstraints = false
+        return airPlayButton
+    }()
+    
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            minimizePlaybackButton.leadingAnchor.constraint(equalTo: self.leadingAnchor),
+            minimizePlaybackButton.topAnchor.constraint(equalTo: self.topAnchor),
+            minimizePlaybackButton.widthAnchor.constraint(equalToConstant: minimizePlaybackButton.frame.width),
+            minimizePlaybackButton.bottomAnchor.constraint(equalTo: self.bottomAnchor),
+            mediaTitleTextLabel.leadingAnchor.constraint(equalTo: minimizePlaybackButton.trailingAnchor, constant: 16),
+            mediaTitleTextLabel.topAnchor.constraint(equalTo: self.topAnchor),
+            mediaTitleTextLabel.trailingAnchor.constraint(equalTo: chromeCastButton.leadingAnchor, constant: 50),
+            chromeCastButton.trailingAnchor.constraint(equalTo: airPlayButton.leadingAnchor, constant: 20),
+            chromeCastButton.topAnchor.constraint(equalTo: self.topAnchor),
+            chromeCastButton.widthAnchor.constraint(equalToConstant: 24),
+            airPlayButton.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: 20),
+            airPlayButton.topAnchor.constraint(equalTo: self.topAnchor),
+            airPlayButton.widthAnchor.constraint(equalToConstant: 24)
+        ])
+    }
+    
+    private func setupViews() {
+        addSubview(minimizePlaybackButton)
+        addSubview(mediaTitleTextLabel)
+        addSubview(chromeCastButton)
+        addSubview(airPlayButton)
+        
+        if UIApplication.shared.statusBarOrientation.isLandscape {
+            mediaTitleTextLabel.isHidden = true
+        }
```

Comment:
do we want that? if yes that needs to be handled on rotation as well

Reply:
This was supposed to be Portrait and I'm not sure, the designs in figma are a bit contradictory. In a few the title is hidden on portrait mode and on others they are there. I just chose one

Reply:
I'll take a look when I test it :)

Reply:
The navigation bar hasnt been added to the MovieViewController as yet, that would be in the next PR